Changelog.md

```diff
@@ -1,3 +1,14 @@
+## 4.0.1 - UNRELEASED
+
+### Changed
+
+-   Moved feedback tab to a dialog which can be opened by going to the about tab
+    and click **Give Feedback**
```

Comment:
This change isn't related to the Performance improvements (I assume should be separate branch). Also this isn't consistent with other apps and I'd consult with @ketile first before doing this change, as it affects all of the apps